src/test/java/test/helm/Helm.java

```diff
@@ -0,0 +1,86 @@
+package test.helm;
+
+import io.vavr.collection.HashMap;
+import org.junit.jupiter.api.TestInfo;
+import test.model.KubeResources;
+import test.model.Product;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * A wrapper around the helm CLI tool, which we use for rendering a chart's templates and capturing the output.
+ */
+public final class Helm {
+    private final TestInfo testInfo;
+
+    public Helm(TestInfo testInfo) {
+        this.testInfo = testInfo;
+    }
+
+    public Path captureHelmTemplateOutput(Product product, Path valuesFile) throws Exception {
+        final var outputFile = getHelmTemplateOutputFile(product);
+        captureHelmTemplateOutput(product, valuesFile, outputFile);
+        return outputFile;
+    }
+
+    private static void captureHelmTemplateOutput(Product product, Path valuesFile, Path outputFile) throws Exception {
+        final var process = new ProcessBuilder()
+                .command("helm", "template",
+                        getHelmReleaseName(product),
+                        getHelmChartPath(product).toString(),
+                        "--debug",
+                        "-n mynamespace",
+                        "--values",
+                        valuesFile.toString())
+                .redirectOutput(outputFile.toFile())
+                .redirectError(outputFile.toFile())
+                .start();
+        final var exitCode = process.waitFor();
+        assertThat(exitCode).isEqualTo(0);
+    }
+
+    public KubeResources captureKubeResourcesFromHelmChart(Product product, Map<String, String> values) throws Exception {
+        final var outputFile = getHelmTemplateOutputFile(product);
+        captureHelmTemplateOutput(product, outputFile, values);
```

Comment:
I just started reviewing this, so I might be missing something.
This line is
```
captureHelmTemplateOutput(product, outputFile, values);
```
while line 29 is
```
captureHelmTemplateOutput(product, valuesFile, outputFile);
```
Is that on purpose, or are the 2nd and third parameters swapped?